src/amp/server/render.tsx

```diff
@@ -58,6 +64,7 @@ export const render = ({ body }: express.Request, res: express.Response) => {
                     articleData={CAPI}
                     nav={extractNAV(CAPI.nav)}
                     analytics={analytics}
+                    experiment={activeTests}
```

Comment:
same :)

Reply:
See above- refactored to improve naming convention